Author: Daniel Baumann <daniel@debian.org>
Description: Sets the gnulib directory.

Index: gnulib/gnulib-tool
===================================================================
--- gnulib.orig/gnulib-tool	2012-04-05 00:45:13.000000000 +0100
+++ gnulib/gnulib-tool	2012-04-05 00:45:31.000000000 +0100
@@ -461,7 +461,7 @@
       * ) self_abspathname=`echo "$self_abspathname" | sed -e 's,/[^/]*$,,'`/"$linkval" ;;
     esac
   done
-  gnulib_dir=`echo "$self_abspathname" | sed -e 's,/[^/]*$,,'`
+  gnulib_dir=/usr/share/gnulib
 }
 
 # func_tmpdir
