Author: Daniel Baumann <daniel@debian.org>
Description: Adjusts shebang.

Index: gnulib/lib/uniname/gen-uninames.lisp
===================================================================
--- gnulib.orig/lib/uniname/gen-uninames.lisp	2014-02-03 00:56:56.000000000 +0000
+++ gnulib/lib/uniname/gen-uninames.lisp	2014-02-03 00:57:09.000000000 +0000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/clisp -C
+#!/usr/bin/clisp -C
 
 ;;; Creation of gnulib's uninames.h from the UnicodeData.txt table.
 ;;; Bruno Haible 2000-12-28
