Author: Daniel Baumann <daniel@debian.org>
Description: Adjusts shebang.

diff -Naurp gnulib.orig/lib/uniname/gen-uninames.lisp gnulib/lib/uniname/gen-uninames.lisp
--- gnulib.orig/lib/uniname/gen-uninames.lisp	2009-04-06 01:40:05.000000000 +0000
+++ gnulib/lib/uniname/gen-uninames.lisp	2009-04-06 01:51:07.000000000 +0000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/clisp -C
+#!/usr/bin/clisp -C
 
 ;;; Creation of gnulib's uninames.h from the UnicodeData.txt table.
 ;;; Bruno Haible 2000-12-28
